frontend/src/store/reduxState.ts

```diff
@@ -1,58 +1,104 @@
+import produce, { Draft } from 'immer';
+
+// As you can see, the redux store can get very populated with lots of types.
+// Generally, it's good to try to segment it by pieces that a component may need
+// This allows your break apart this stuff and avoid one large file.
+
+// These can all go in 'types'
+export interface IEmote {
+  ownerUsername: string;
+  command: string;
+}
```

Comment:
I believe this type (IEmote) is also something that should/could be shared between the frontend and backend.